Civi/Api4/Action/Entity/Get.php

```diff
@@ -25,7 +25,9 @@ class Get extends \Civi\Api4\Generic\BasicGetAction {
    */
   protected function getRecords() {
     $entities = [];
-    foreach (explode(PATH_SEPARATOR, get_include_path()) as $path) {
+    $extensionSystem = \CRM_Extension_System::singleton()->getFullContainer();
```

Comment:
Oh, hang on, this seems to be returning every extension even if they are disabled. Is there a simple way to get the path of every enabled extension?

Reply:
Looks like I have to use `\CRM_Extension_Manager::getStatus()` and then filter the list by `STATUS_INSTALLED`??